code/modules/art/paintings.dm

```diff
@@ -372,7 +372,7 @@
 	framed_offset_y = 5
 
 /obj/item/canvas/twentyfour_twentyfour
-	name = "canvas (AI Universal Standard)"
+	name = "canvas (24x24) (AI Universal Standard)"
```

Comment:
This looks a bit verbose, and perhaps should be simply renamed to "canvas (24x24)". It's unlikely that people will draw on it because they want to give the AI something to display on their screen.

Reply:
I'm pretty sure plenty of people paint stuff with intent to use them next time thye're AI, there's many AI players that have some static painting as their core from my experience.

Reply:
Well, that's also true.